client/src/containers/profile/config.ts

```diff
@@ -2,21 +2,10 @@ import { ActiveTabId } from './logic/models';
 
 export const tabName: Record<ActiveTabId, string> = {
 	[ActiveTabId.Stats]: 'Stats',
-	[ActiveTabId.Challenge]: 'Challenge',
+	[ActiveTabId.Tasks]: 'Tasks',
```

Comment:
This tab should be "Challenge" as we have in the figma design. It's a mistake from mocks in the trello ticket

![image](https://user-images.githubusercontent.com/55874288/128297632-a30b7dd8-829c-4b24-ae87-5fc3bab113d9.png)